test/parallel/test-stream-pipe-unpipe-streams.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+const { Readable, Writable } = require('stream');
+
+const source = Readable({read: () => {}});
+const dest1 = Writable({write: () => {}});
+const dest2 = Writable({write: () => {}});
+
+source.pipe(dest1);
+source.pipe(dest2);
+
```

Comment:
Nit: probably unrelated but maybe it makes sense to assert that `source._readableState.pipes` contains both `dest1` and `dest2` now.